packages/noya-geometry/src/line.ts

```diff
@@ -42,3 +43,22 @@ export function isPointInLine(point: Point, line: [Point, Point]) {
 
   return Math.abs(d1 + d2 - lineLength) < 0.5;
 }
+
+// TODO: Not really accurate, but works for now.
+export function isPointInCircle(
```

Comment:
We should rename this `isPointOnCircumference`, since the current name implies that any point within the circle would give `true`